internal/backend/xattr_linux.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Francisco Souza. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
Can you rename this file to `xatter_unix.go` and delete `xattr_darwin.go`? With the right build tags we should be able to do it:

```suggestion
//go:build !windows
// +build !windows

```

Reply:
Done.